src/containers/ConnectorManagement.jsx

```diff
@@ -8,6 +8,8 @@ import Notifier from '../components/Notifier'
 const AUTHORIZED_DATATYPE = require('config/datatypes')
 const isValidType = (type) => AUTHORIZED_DATATYPE.indexOf(type) !== -1
 
+import stylesLoginForm from '../styles/accountLoginForm'
```

Comment:
In every other component, we use `styles` to store component's related styles. It seems that `styles` is already used here, so it's pretty sure that if we need another style here, it's because it is not the right place to use it.

However, we don't need `styles` to provide a selector (see below).